### DISCUSSION TITLE
GPT-4o-mini

### DISCUSSION THREAD

**Discussion by wallacenagashima** [langgenius/dify#6751] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I created a chatbot and it was using GPT-4o. I switched to GPT-4o-mini but it continues to charge the GPT-4o API. Has anyone ever experienced this?

<img width="391" alt="Captura de Tela 2024-07-28 às 16 27 25" src="https://github.com/user-attachments/assets/894139fa-0ed0-4b62-9808-41e102fa952c">

**Comment by crazywoola** [comment]
The new models only apply for the new conversation. So the legacy conversation will continue use the old ones.
